packages/crayons-core/src/components/file-uploader-2/file-uploader-2.tsx

```diff
@@ -798,20 +801,27 @@ export class FileUploader {
             {!this.errorText.length ? (
               <div
                 class='file-uploader__body__dropzone__center__clickable__text'
+                part='fw-file-uploader-text'
                 innerHTML={
                   this.text || TranslationController.t('fileUploader2.text')
                 }
               ></div>
             ) : (
-              <div class='file-uploader__body__dropzone__center__clickable__error'>
+              <div
+                class='file-uploader__body__dropzone__center__clickable__error'
+                part='fw-file-uploader-error'
+              >
                 {this.errorText}.{' '}
                 <span class='highlight'>
                   {TranslationController.t('fileUploader2.retry')}
                 </span>
               </div>
             )}
             {this.description && (
-              <div class='file-uploader__body__dropzone__center__clickable__description'>
+              <div
+                class='file-uploader__body__dropzone__center__clickable__description'
+                part='fw-file-uploader-desc'
```

Comment:
Please add the details of the `parts` in the documentation as well. Part Name and its description

Reply:
done

Reply:
@Tahseen-Ibrahim Am unable to see any changes in the documentation. Can you please check if you have pushed the changes.

Reply:
yes pushed